### PR TITLE
fix: The abnormal problem of tooltip on the page.

### PR DESCRIPTION
**Pull request by JeffreyMa597**
This PR is in order to resolve the issue when we hover an element which has a tooltip, it will be a moment delay.
**Before:**

1、
image
2、
image

**After:**
image

Ref: [Magickbase/godwoken_explorer/issues/#836](https://github.com/Magickbase/godwoken_explorer/issues/836)

### CODE REVIEW

File: styles/tooltip.scss [hunk 1]
```diff
@@ -0,0 +1,60 @@
+@mixin relative-inline-block {
+  position: relative;
+  display: inline-block;
+}
+@mixin tooltip-content-title {
+  content: attr(data-tooltip);
+}
+
+$tooltip-border-color: #333;
```
Comment (Keith-CY) [inline styles/tooltip.scss:9]: css variables could be used instead of sass variables

**Comment by Keith-CY** [comment]
Is this PR ready for review?

**Comment by Keith-CY** [comment]
The error message in vercel is as follow

```

[09:47:10.206] Failed to compile.
[09:47:10.206] 
[09:47:10.207] ./components/AccountOverview/index.tsx:151:22
[09:47:10.207] Type error: Element implicitly has an 'any' type because expression of type '"address"' can't be used to index type 'Variables'.
[09:47:10.207]   Property 'address' does not exist on type 'Variables'.
[09:47:10.207] 
[09:47:10.252] Error: Command "npm run build" exited with 1
```

**Comment by JeffreyMa597** [comment]
> The error message in vercel is as follow
> 
> ```
> 
> [09:47:10.206] Failed to compile.
> [09:47:10.206] 
> [09:47:10.207] ./components/AccountOverview/index.tsx:151:22
> [09:47:10.207] Type error: Element implicitly has an 'any' type because expression of type '"address"' can't be used to index type 'Variables'.
> [09:47:10.207]   Property 'address' does not exist on type 'Variables'.
> [09:47:10.207] 
> [09:47:10.252] Error: Command "npm run build" exited with 1
> ```

It seems to be some typescript errors in our old component pages. I've killed them.

**Comment by JeffreyMa597** [comment]
> The error message in vercel is as follow
> 
> ```
> 
> [09:47:10.206] Failed to compile.
> [09:47:10.206] 
> [09:47:10.207] ./components/AccountOverview/index.tsx:151:22
> [09:47:10.207] Type error: Element implicitly has an 'any' type because expression of type '"address"' can't be used to index type 'Variables'.
> [09:47:10.207]   Property 'address' does not exist on type 'Variables'.
> [09:47:10.207] 
> [09:47:10.252] Error: Command "npm run build" exited with 1
> ```

I've double-check our project, I found this property of tsconfig may cause this problem also after I changed it to be `true`. So, I set it back to `false`.
image

**Comment by Keith-CY** [comment]
Build failed due to
```


Failed to compile.
--
23:18:27.207 |  
23:18:27.208 | ./components/ERC20TransferList/index.tsx:42:48
23:18:27.209 | Type error: Cannot find name 'Nullable'.
23:18:27.209 |  
23:18:27.209 | 40 \| }
23:18:27.210 | 41 \|
23:18:27.210 | > 42 \| interface AccountTransferListVariables extends Nullable<Cursor> {
23:18:27.210 | \|                                                ^
23:18:27.210 | 43 \|   address: string
23:18:27.211 | 44 \| }
23:18:27.211 | 45 \|
23:18:27.255 | Error: Command "npm run build" exited with 1


```

**Comment by Keith-CY** [comment]
Conflicted

**Comment by JeffreyMa597** [comment]
> Conflicted

Resolved.

**Comment by Keith-CY** [comment]
CI blames that lock file is inconsistent with package.json

**Review by qiweiii — APPROVED** [review]

File: components/AccountOverview/index.tsx [hunk 2]
```diff
@@ -164,11 +164,11 @@ export const fetchDeployAddress = (variables: { eth_hash: string }) =>
 
 export const fetchSourcifyStatus = (address: string) =>
   client.request<{ sourcify_check_by_addresses: { status: string | null } }>(checkSourcify, { address }).then(data => {
-    return data.sourcify_check_by_addresses[0].status
+    return data.sourcify_check_by_addresses.status
```
[the remark this answers is not in the document]
Comment (JeffreyMa597) [reply components/AccountOverview/index.tsx:167]: I also think the type seems to be a little problem. I'll change it.

File: components/AccountOverview/index.tsx [hunk 3]
```diff
@@ -231,24 +231,56 @@ const AccountOverview: React.FC<AccountOverviewProps & { refetch: () => Promise<
...
+    })
+
   return (
     <div className={styles.container} data-account-type={account.type}>
       {account.type === GraphQLSchema.AccountType.MetaContract ? (
         <MetaContract {...(account.script as MetaContract['script'])} />
       ) : null}
       {account.type === GraphQLSchema.AccountType.EthUser ? (
-        <User nonce={account.nonce} isLoading={isOverviewLoading} />
+        <User nonce={account.nonce} isLoading={!!isOverviewLoading} />
```
[the remark this answers is not in the document]
Comment (Keith-CY) [reply components/AccountOverview/index.tsx]: The pr review could be `resolved` if advice is adopted and better to leave a link to the new code

**Comment by FrederLu** [comment]
Before：
image


Now：
image
After the modification, it is displayed normally on the mobile phone.

@Keith-CY It's just that the format of the tooltip displayed is now 4 lines, before it was 2 lines (does this need to be adjusted).

**Comment by Keith-CY** [comment]
> Before： image
> 
> Now： image After the modification, it is displayed normally on the mobile phone.
> 
> @Keith-CY It's just that the format of the tooltip displayed is now 4 lines, before it was 2 lines (does this need to be adjusted).

I think it's fine if the content is displayed clearly.

**Review by FrederLu — APPROVED** [review]

**Pull request merged**